packages/migrate/src/__tests__/MigrateDev.test.ts

```diff
@@ -502,7 +465,8 @@ describe('sqlite', () => {
 
       We need to reset the SQLite database "dev.db" at "file:dev.db"
       
-      You may use --allow-reset to drop the database. All data will be lost.
+      You may use prisma migrate reset to drop the development database.
```

Comment:
```suggestion
      You may use the `prisma migrate reset` command to drop the development database.
```
I'd wrap it in some quotes to clarify it's a command

Reply:
The command has already been wrapped in red:
```ts
      process.stdout.write(
        '\n' +
          `You may use ${red('prisma migrate reset')} to drop the development database.\n` +
          `${bold(red('All data will be lost.'))}\n`,
      )
```

The stdout capture in the test is just clearing all the coloring, therefore it does not show up in the expected output.